<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>month_total</class>
 <widget class="QDialog" name="month_total">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查看每月总收支</string>
  </property>
  <widget class="QLabel" name="label_hint1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>126</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>该月总支出：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hint2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>186</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>该月总收入：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_expense">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>126</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_income">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>93</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="labelMonth">
    <property name="text">
     <string>月份</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_month">
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="label_hint2_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>244</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>该月净收入：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_balanced">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>245</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <zorder>back</zorder>
  <zorder>splitter_2</zorder>
  <zorder>label_hint1</zorder>
  <zorder>label_hint2</zorder>
  <zorder>label_expense</zorder>
  <zorder>label_income</zorder>
  <zorder>label_hint2_2</zorder>
  <zorder>label_balanced</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
